Package.swift

```diff
@@ -4,25 +4,30 @@
 import PackageDescription
 
 let package = Package(
-    name: "starknet",
+    name: "Starknet",
```

Comment:
Just a nit but maybe it should be `StarkNet` to match the brand?

Reply:
We used this convention in starknet-jvm. I think the reason for that was, that when using longer names, like StarkNetChainId, it could seem like Stark and Net are different things. So we preferred to use Starknet.